GoogleUtilities/Reachability/GULReachabilityChecker.m

```diff
@@ -22,7 +22,7 @@
 #import <GoogleUtilities/GULReachabilityChecker.h>
 
 static GULLoggerService kGULLoggerReachability = @"[GULReachability]";
-
+#if !TARGET_OS_WATCH
```

Comment:
Looks like we can replace this the functionality with https://developer.apple.com/documentation/watchconnectivity/wcsession/1615683-reachable?language=objc

Reply:
is the reachability class to test an iPhone is connected to internet? The WCSession seems to test if a watch is connected to a companion phone and i don't know if this is 1:1 matching functionality. I wonder if I should work with the reachability owner to see how we can incorporate watch API in it. 

This is one of my concern in GoogleUtilities and FireLog pod, where I just comment out code because for messaging we don't need it. But I guess to make each pod function in watchOS, it's going to take more work.